src/main/webapp/scripts/massedit.js

```diff
@@ -33,7 +33,7 @@ HBT.MassEdit = HBT.MassEdit || (function(){
 					ids.push(id);
 				}
 			});
-			form.action = "/tb/do/ShowDailyReport?task=MassDelete&ids=" + ids.join(",");
+			form.action = "../do/ShowDailyReport?task=MassDelete&ids=" + ids.join(",");
```

Comment:
Warum ändert sich der Pfad hier?

Reply:
Weil das Javascript von genau einer JSP-Datei geladen wird, welches sich nun in einem Unterordner befindet. Pfade sind nun relativ zum Pfad der JSP-Seite.